pkg/datarepair/queue/queue.go

```diff
@@ -60,14 +62,20 @@ func (q *Queue) Enqueue(qi *pb.InjuredSegment) error {
 	if err != nil {
 		return queueError.New("error adding injured seg to queue %s", err)
```

Comment:
line 63. If we are unable to put the injured segment into the queue, 1. should we handle it to find out why we are unable to put into the queue and try a recovery mechanism to find out why and try again to put into the queue? 2. if is the first segment that we are trying to put into the queue and it failed, should we issue a cond.Signal()?

Reply:
1. Probably not, since we should assume that the checker will eventually pick it up again.
2. I'm not quite sure what that would accomplish.

Reply:
agreed with (1).  (2)please ignore...My bad I skipped the 'return'. ignore comment (2)